cmd/cli/install.go

```diff
@@ -91,6 +91,8 @@ var possibleEnvoyLogLevels = []string{"trace", "debug", "info", "warning", "warn
 // Its value is initialized at build time.
 var chartTGZSource string
 
+var log = logger.NewPretty("osm-install")
```

Comment:
I don't know that we need a logger in the CLI. Why not just print to stdout/stderr?